db/seeds.rb

```diff
@@ -7,14 +7,17 @@
 #   Character.create(name: 'Luke', movie: movies.first)
 
 puts "Cleaning up the database..."
-Reason.destroy_all
-Comment.destroy_all
-Reason.destroy_all
 Point.destroy_all
+PointComment.destroy_all
 Service.destroy_all
+ServiceComment.destroy_all
 Case.destroy_all
+CaseComment.destroy_all
 Topic.destroy_all
+TopicComment.destroy_all
+Document.destroy_all
 User.destroy_all
+Version.destroy_all
```

Comment:
These changes are needed to enable the `db:seed` command not to throw an error when it tries to empty a table that doesn't exist. It is separate from the installation process, but required for the larger setup process.

Reply:
Ok then, I'd prefer to have this in a separate merge or having another title, but it's ok anyway.